components/button-dropdown/src/overrides/heading.js

```diff
@@ -1,25 +1,27 @@
 /** @jsx jsx */
 
-import { jsx, useBrand } from '@westpac/core';
+import { jsx, useBrand, getLabel } from '@westpac/core';
 import { Heading } from '@westpac/heading';
 
 const ButtonDropdownHeading = ({ state: { tag }, ...rest }) => (
-	<Heading size={9} tag={tag} {...rest} />
+	<Heading size={8} tag={tag} {...rest} />
 );
 
 const headingStyles = () => {
-	const { COLORS } = useBrand();
+	const { COLORS, TYPE } = useBrand();
 
 	return {
-		margin: '1.5rem -0.375rem 1.125rem -0.375rem',
-		padding: '0.4375rem',
-		backgroundColor: COLORS.background,
-		color: COLORS.hero,
-		fontWeight: '500', // what is the correct way to do this??
-		// fontSize: '0.75rem',
+		'&&&': {
```

Comment:
Why are we adding this? Are there specificity issues?

Reply:
yes specificity needs to be overridden from the Heading component